ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -427,17 +427,34 @@ objects:
         terraform apply -auto-approve -var 'bootstrap_complete=true' -no-color &
         wait "$!"
 
+        function approve_csrs() {
+          while true; do
+            if [[ ! -f /tmp/install-complete ]]; then
```

Comment:
Am I reading this wrong or does this wait for the install-complete file to exist before approving the CSRs? 

I was expecting something like the following.
```
function approve_csrs() {
  while [[ ! -f /tmp/install-complete ]]; do
    sleep 15;
    oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs --no-run-if-empty oc adm certificate approve
  done
}
```

Reply:
Oh, correct, this won't ever finish, as CSR should be approved earlier

Reply:
What are we gaining by having an `if` statement that does a continue in one branch and a break in the other rather than moving the check up to the `while` statement?